### ISSUE TITLE
Humans deform when quick stopping if collision is dynamic or continous

### ISSUE THREAD

**Issue by DindinYT37** [studio-minus/ppg-bugs-and-requests#1133]
**Describe the bug**
If, for example, you yeet an indestructible human (so all of his limbs don't immediately fall off) or an android (sometimes work) with high speed into the air and suddenly stop him with for example, the mouse they "deform".

**Clearly describe how to reproduce the bug**
1. Set Collision quality to Dynamic (happens sometimes) or Continuous
2. Throw indestructible human (or android) into the air with high speed
3. Catch him in the air, sometimes it only happens in slow-mo, sometimes only in normal speed - I recommend trying both.
4. Repeat a few times until you get a result, if it doesn't happen after repeated trying, I can make a video of it.

**Mod list**
Happens in Vanilla too.

**System info (please complete the following information)**
 - Operating system: Windows 10
 - Game version: 1.23.7
 - GPU: Nvidia GeForce GTX 1070
 - CPU: AMD Ryzen 2700X

**Additional notes**
Look what they did to my boy.
![image](https://user-images.githubusercontent.com/64973205/163676808-8ca89e64-c6c6-46e9-a685-018791f7e6f6.png)

**Comment by basedkit** [comment]
i will also add that this is a extremely common bug thats been around for a long time and happens the most when you over drag a human in pause mode or if you move them to fast then pause and drag them.

**Comment by DindinYT37** [comment]
> i will also add that this is a extremely common bug thats been around for a long time and happens the most when you over drag a human in pause mode or if you move them to fast then pause and drag them.

Yeah, I was wondering why nobody reported it.

**Comment by mestiez** [comment]
this happens because its switching between physics-dragging and pause-mode-dragging. im not sure what im supposed to do here.

I could have it not automatically switch between dragging modes if the player is still dragging

**Comment by DindinYT37** [comment]
> this happens because its switching between physics-dragging and pause-mode-dragging. im not sure what im supposed to do here.
> 
> I could have it not automatically switch between dragging modes if the player is still dragging

It also happens when not pausing, though rarer it does happen.

**Comment by mestiez** [comment]
I've never encountered that

**Comment by basedkit** [comment]
> I've never encountered that

Get a plane and make it move relatively fast, then pause the game and drag the human

**Comment by DindinYT37** [comment]
> > I've never encountered that
> 
> Get a plane and make it move relatively fast, then pause the game and drag the human

That's pausing the game though-

**Comment by basedkit** [comment]
> > > I've never encountered that
> > 
> > 
> > Get a plane and make it move relatively fast, then pause the game and drag the human
> 
> That's pausing the game though-

Yep, that’s the easiest way to recreate it

**Comment by DindinYT37** [comment]
> > > > I've never encountered that
> > > 
> > > 
> > > Get a plane and make it move relatively fast, then pause the game and drag the human
> > 
> > That's pausing the game though-
> 
> Yep, that’s the easiest way to recreate it

Yeah but the whole point was that I said it also happens when not pausing and zooi said "I've never encountered that".

**Comment by basedkit** [comment]
> 

Ah sorry I misunderstood

**Comment by ppgplayer** [comment]
G forces mayb?

**Comment by DindinYT37** [comment]
> G forces mayb?

As far as I know, that's just causing the damage.

**Comment by Drogo-man** [comment]
![download](https://user-images.githubusercontent.com/102727246/164012268-e8b90084-b1a1-4d6c-aec5-cff2cc14b7cc.jpg)

**Comment by mestiez** [comment]
I can't reproduce this reliably*??
